src/sentry/tasks/summaries/daily_summary.py

```diff
@@ -209,3 +229,44 @@ def build_summary_data(
         fetch_key_performance_issue_groups(ctx)
 
     return ctx
+
+
+def build_top_projects_map(context: OrganizationReportContext, user_id: int):
+    """
+    Order the projects by which of the user's projects have the highest error count for the day
+    """
+    user_projects_context_map: dict[int, DailySummaryProjectContext] = {}
+    projects_context_map = cast(dict[int, DailySummaryProjectContext], context.projects_context_map)
+    for project in context.project_ownership[user_id]:
+        user_projects_context_map[project] = projects_context_map[project]
+
+    projects_by_error_total = {
+        project_id: context.total_today for project_id, context in user_projects_context_map.items()
+    }
+    top_projects = [
+        k
+        for k, v in sorted(projects_by_error_total.items(), key=lambda item: item[1], reverse=True)
+    ]
+    top_projects_context_map = {}
+    # for now, hard code to the top 2 projects
+    for project in top_projects[:2]:
+        project_context = user_projects_context_map[project]
+        top_projects_context_map[project] = project_context
+
+    return top_projects_context_map
+
+
+def deliver_summary(ctx: OrganizationReportContext, users: list[int]):
+    # TODO: change this to some setting for daily summary
```

Comment:
I think for the internal MVP this is ok, but if/when we release to customers we should create a new setting for it.